.github/workflows/find-inactive-collaborators.yml

```
@@ -13,5 +13,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - run: tools/find-inactive-collaborators.mjs '1 year ago'
+      - uses: actions/checkout@v2
+    
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
```

For the sake of consistency across our workflow files, you could add this bit above:
```yaml
env:
  NODE_VERSION: 16.x
```
And re-use the same step as others workflows:
```suggestion
      - name: Use Node.js ${{ env.NODE_VERSION }}
        uses: actions/setup-node@v2
        with:
          node-version: ${{ env.NODE_VERSION }}
```